app/api/groups/discard_code.go

```diff
@@ -17,7 +17,7 @@ import (
 //
 //
 //   The authenticated user should be a manager of `group_id` with `can_manage` >= 'memberships',
-//   otherwise the 'forbidden' error is returned.
+//   otherwise the 'forbidden' error is returned. If the group is a user group, the 'forbidden' error is returned as well.
```

Comment:
The term "user group" that you use everywhere is a bit misleading as a class is "user group" (in the sense "group of users"). I think it will be clearer if you just write "If the group is a user, ..."

Reply:
fixed